apps/src/levelbuilder/skills/SkillEvaluationSettings.tsx

```diff
@@ -50,7 +52,10 @@ const SkillEvaluationSettings: React.FC<Props> = ({
           size="s"
         />
       </p>
-      <AccuracyCheck levelId={levelId} />
+      <AccuracyCheck
+        levelId={levelId}
+        questionAiEvaluatable={levelType === 'FreeResponse'}
```

Comment:
I don't think we necessarily want to limit this to 'Free Response' since this would work for coding levels too...